tasks/models/user.js

```diff
@@ -24,6 +24,11 @@ export function isUserExist(id) {
     return storage.has(id);
 }
 
+export function getUserByLogin(login) {
+    const userByLogin = Array.from(storage.values()).filter(user => user.login === login);
```

Comment:
Consider using `[...].find()` method instead of filter. This way you won't need to get the first element of the resulting array on the next line.